python/src/opendp/_lib.py

```diff
@@ -26,7 +26,10 @@ def _load_library():
         lib_dir = Path(__file__).parent / ".." / ".." / ".." / 'rust' / 'target' / build_dir  # pragma: no cover
 
     if lib_dir.exists():
-        lib_dir_file_names = [p for p in lib_dir.iterdir() if p.suffix in {".so", ".dylib", ".dll", ".pyd"}]
+        lib_dir_file_names = [
+            p for p in lib_dir.iterdir()
+            if p.suffix in {".so", ".dylib", ".dll", ".pyd"}
+            and p.stem == 'libopendp']
```

Comment:
If I recall correctly, on windows the stem doesn't include "lib". I think this will work:
```suggestion
            and p.stem.replace("lib", "") == "opendp"]
```

Reply:
Thanks!